x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/privilege_monitoring/privilege_monitoring_data_client.ts

```diff
@@ -115,6 +138,8 @@ export class PrivilegeMonitoringDataClient {
       this.opts.telemetry?.reportEvent(PRIVMON_ENGINE_INITIALIZATION_EVENT.eventType, {
         duration,
       });
+      // sync all index users from monitoring sources
+      await this.syncAllIndexUsers();
```

Comment:
I am not sure we need to sync on init, will there ever be users in there? Could we wait for the scheduled sync?